packages/presets/src/ai/chat-panel/chat-panel-input.ts

```diff
@@ -294,9 +307,18 @@ export class ChatPanelInput extends WithDisposable(LitElement) {
           >
             ${ImageIcon}
           </div>
-          <div @click="${this.send}" class="chat-panel-send">
-            ${ChatSendIcon}
-          </div>
+          ${this.status === 'transmitting'
+            ? html`<div
+                @click=${() => {
+                  this.abortController?.abort();
+                  this.updateStatus('idle');
```

Comment:
I think this is not necessary because when stream closes the status will be change to `success`. BTW what's the difference between idle vs success?

Reply:
Indeed, this is redundant. 
the idle only indicates that chat panel are first loaded into the view.